deploy/examples/toolbox-operator-image.yaml

```diff
@@ -22,6 +22,7 @@ spec:
         app: rook-ceph-tools-operator-image
     spec:
       dnsPolicy: ClusterFirstWithHostNet
+      serviceAccountName: rook-ceph-default
```

Comment:
Does that mean toolbox deployment should also use the default service account created by rook, instead of k8s default service account?

Reply:
Yes, toolbox needs to use the service account, it's updated below as well.